cql3/functions/functions.cc

```diff
@@ -382,6 +385,32 @@ functions::get(data_dictionary::database db,
     return std::move(compatibles[0]);
 }
 
+template<typename F>
```

Comment:
Please add a constraint (could by as simple as std::same_as<X> || std::same_as<Y>. Fully generic templates mean that an unsuspecting reader has to go and look at the callers, or reverse engineer the function to understand what the F F is.

Reply:
added concept of describable

Reply:
I don't see it

Reply:
Reading the description, it's get_functions_of_class<>.